app/models/artist.rb

```diff
@@ -42,6 +44,11 @@ def merge(other)
       ta.update(artist_id: id)
     end
 
+    # rubocop:disable Rails/SkipsModelValidations
+    # Since we only update the item_id, there aren't any validations that could fail
+    other.playlist_items.update_all(item_id: id)
+    # rubocop:enable Rails/SkipsModelValidations
+
```

Comment:
Doesn't this mean there could be duplicates in a playlist? I guess that's not a failed validation, but is this something we want in this case?

Reply:
we validate the uniqueness of an item_id within a playlist (both with validations and with an index) - but I'm not sure what would happen here. Probably an error? 

I'll check + add a test to see that the merging will always succeed (and do the same for tracks)